### DISCUSSION TITLE
esxi8 richtiges guest OS

### DISCUSSION THREAD

**Discussion by Novell2** [OpenCCU/OpenCCU#2423] [🇩🇪 Fragen&Antworten]
Hallo es ist zwar nur ein schönheitsfehler, wenn ich in vsphere die VM kontrolliere, vergrössere bekomme ich die Meldung das es nicht das richtige Guest OS wäre. Vor einige Zeit hatte ich RaspbeeryMatic mit OVA installiert. In welche Linux Kategorie gehört diese 64Bit Version?
Zur Auswahl habe ich 77 Möglichkeiten.
[esxi8-Guest OS Linux 64bit.xlsx](https://github.com/jens-maus/RaspberryMatic/files/12511370/esxi8-Guest.OS.Linux.64bit.xlsx)
Besten Dank!
Gruss Novell1

**Comment by jens-maus** [comment]
Das versuchst du doch gerade hier zu klären:

https://homematic-forum.de/forum/viewtopic.php?p=776607#p776607

Insofern braucht es keinen doppelpost und erst recht nicht mit ekligem Excel File als Anhang ;)
